constants/fokabotCommands.py

```diff
@@ -123,7 +123,17 @@ async def alert(fro: str, chan: str, message: list[str]) -> None:
     if not (msg := " ".join(message).strip()):
         return
 
+    userID = await userUtils.getID(fro)
     await streamList.broadcast("main", serverPackets.notification(msg))
+    await rap_logs.send_rap_log(
+        userID,
+        f"has sent an alert to all users: '{msg}'",
+    )
+    await rap_logs.send_rap_log_as_discord_webhook(
+        message=f"[{fro}](https://akatsuki.gg/u/{userID}) ({userID}) has sent an alert to all users:```{msg}```",
```

Comment:
i think some messages could accidentally get treated as a language specifier here
```suggestion
        message=f"[{fro}](https://akatsuki.gg/u/{userID}) ({userID}) has sent an alert to all users:```\n{msg}```",
```